### ISSUE TITLE
Sign in issues

### ISSUE THREAD

**Issue by zooz89** [jellyflix-app/jellyflix#138]
I downloaded Jellyflix excited to try out its features but unfortunately was unable to sign in to my Server. Upon filling out the relevant information and pressing the sign in button I was met with nothing. No error messages, no loading screens, no login. I was able to log in using the default Jellyfin client using the same information, so it was not an error there. I look forward to trying your product in the future. I hope you are able to resolve this issie :)

**Comment by sevenrats** [comment]
Hi can you produce logs from the server of the failed login event?

**Comment by zooz89** [comment]
Unfortunately it is a server hosted by my friend so I don’t have the best
quality access to the logs haha but here is what he sent me. It looks like
the issue is the 6:40 log stating a username error. But I was able to use
the official Jellyfin client with the exact same login details.

**Comment by jdk-21** [comment]
Can you provide information about the OS, the Jellyflix and the Jellyfin version you are using?

**Comment by navneetlal** [comment]
I'm encountering the same problem. There are no error logs in my Jellyfin instance, only these four info logs appear when I attempt to log in.

**jellyfin:** 10.9.11
**OS:** macOS 15.1

```bash
[10:33:51] [INF] [45] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for <username> has succeeded.
[10:33:51] [INF] [45] Emby.Server.Implementations.Session.SessionManager: Current/Max sessions for user <username>: 2/0
[10:33:51] [INF] [45] Emby.Server.Implementations.Session.SessionManager: Logging out access token <token>
[10:33:51] [INF] [45] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user <token>
```

**Comment by jdk-21** [comment]
@navneetlal The macOS (dmg) 0.10.X version has an issue accessing the keychain. You can try using the latest prerelease version or the App Store version

**Comment by jvasile** [comment]
I have the same issue with the 0.10.0 release running on Debian Testing.  My Jellyfin server logs similarly show me logging in:

```

[2024-11-16 21:50:12.688 +00:00] [INF] [125] Jellyfin.Server.Implementations.Users.UserManager: Authentication request for "james" has succeeded.
[2024-11-16 21:50:12.690 +00:00] [INF] [125] Emby.Server.Implementations.Session.SessionManager: Current/Max sessions for user "james": 2/0
[2024-11-16 21:50:12.691 +00:00] [INF] [125] Emby.Server.Implementations.Session.SessionManager: Logging out access token "0331f3db6929451b86dee12b40ab3cff"
[2024-11-16 21:50:12.708 +00:00] [INF] [125] Emby.Server.Implementations.Session.SessionManager: Creating new access token for user 706159fe-df52-49c0-b878-233a41458a8b
```

But I get no visible change in the application state after hitting login.

I see an error on the terminal: `** (jellyflix:183208): WARNING **: 16:50:12.769: libsecret_error: \xd4Y)Y\xabU`

And note that when I run from a fresh git checkout, I have keyring issues (see #145 for details).

Jellyfin version 10.9.10, running on a Debian Testing box on my LAN.

**Comment by nzerinto** [comment]
Yep, same here. Just downloaded the app, input server and user details, clicked "Login"......aaaaaaand nothing.

Even deleted the app and redownloaded, as the first time I'd selected "no" to the "_allow app to access local networks_" question. So the next time I selected "yes", but no difference.

**Comment by sevenrats** [comment]
@jdk-21 this is getting kinda interesting actually. we now have keyring failure reports from all 3 platforms.  somebody should probably dig into flutter secure storage when we have a sec i wonder if they dropped a bad version or maybe we missed an important change.

**Comment by nzerinto** [comment]
@sevenrats I just figured my issue mirrors exactly what @zooz89 experienced, in that I downloaded the app to connect to my Jellyfin server, and it doesn't do anything after inputting server/login details and hitting the "_Login_" button.
Have no problem connecting with the native Jellyfin app nor Streamyfin. Host on Windows 11, running ver 10.10.1

**Comment by sevenrats** [comment]
yeah somebody is just going to have to dig in too.  the macos reference is not from the OP so this issue is just in the weeds and is going to need a bunch of handholding.  thats life I guess.

**Comment by jdk-21** [comment]
flutter_secure_storage recently changed the maintainer if I'm not mistaken. 
My understanding is that getting the encryption key just doesn't complete. Maybe a timeout for this function is a quick fix. Then we can explore longer term solutions.

**Comment by jdk-21** [comment]
The new stable release should fix this issue. Please reopen the issue, if the problem still persists.
